### DISCUSSION TITLE
تغییر تمپلیت clash در مرزبان

### DISCUSSION THREAD

**Discussion by ghost** [Gozargah/Marzban#398] [Show and tell]
سلام
من یه تمپلیت کلش برای استفاده خودم اماده کردم که توش سایت های ایرانی مستقیم وصل میشه و ... که گفتم با شما به اشتراک بذارم.

[لینک گیت هاب تمپلیت من](https://github.com/oXIIIo/marzban-template/tree/master/clash)

# ویژگی ها
- وصل شدن به سریع ترین کانفیگ
- وصل شدن به اولین کانفیگ سالم
- لودبالانس
- وصل شدن مستقیم به وب‌سایت‌های ایرانی هنگامی که فیلترشکن روشن است
- بلاک کردن تبلیغات
و ...

# مراحل نصب
1. دانلود فایل template
```sh
sudo wget -N -P /var/lib/marzban/templates/clash/ https://raw.githubusercontent.com/oXIIIo/marzban-template/master/clash/default.yml
```
2. دستورات زیر رو تو ترمینال سرورتون بزنید:
```sh
echo 'CUSTOM_TEMPLATES_DIRECTORY="/var/lib/marzban/templates/"' | sudo tee -a /opt/marzban/.env
echo 'CLASH_SUBSCRIPTION_TEMPLATE="clash/default.yml"' | sudo tee -a /opt/marzban/.env
```
یا مقادیر زیر رو در فایل `.env` در پوشه `/opt/marzban` قرار بدین
```sh
CUSTOM_TEMPLATES_DIRECTORY="/var/lib/marzban/templates/"
CLASH_SUBSCRIPTION_TEMPLATE="clash/default.yml"

3. ری استارت مرزبان
```sh
marzban restart
```
## بروزرسانی
برای بروزرسانی تمپلیت فقط کافیست مرحله 1 را تکرار کنید.

# لینک دانلود کلش متا
- Android:
   - [Clash. Meta for Android](https://github.com/MetaCubeX/ClashMetaForAndroid/releases)
- iOS:
  - [MFI](https://t.me/meta_for_ios)
- Windows：
  - [clash-verge](https://github.com/zzzgydi/clash-verge)
  - [ClashMetaForWindows_Mini](https://github.com/kogekiplay/ClashMetaForWindows_Mini)
  - [clashN](https://github.com/2dust/clashN)
  - [v2rayN](https://github.com/2dust/v2rayN)
  - [Clash.Mini](https://github.com/MetaCubeX/Clash.Mini)
- Mac OS：
  - [clash-verge](https://github.com/zzzgydi/clash-verge)
  - [ClashX.Meta](https://github.com/MetaCubeX/ClashX.Meta)
- Linux：
  - [clash-verge](https://github.com/zzzgydi/clash-verge)

**Comment by AdrianChasee** [comment]
سلام وقت بخیر

من این تمپلیت رو توی سرور اضافه کردم ولی تغییری نکرده کانفیگ ها
میشه راهنمایی کنید لطفا؟

**Reply by ghost** [reply]
سلام عکس فایل .env رو بفرستید خط های اخر.

```
nano /opt/marzban/.env
```

**Comment by wikm360** [comment]
سلام و درود 
شرمنده اون کلش متا برای ios که گذاشتین به نام MFI شما خودتون استفاده کردین ؟ خوبه ؟

**Comment by DeAref** [comment]
با این ارور مواجه میشه کِلش 
![image](https://github.com/Gozargah/Marzban/assets/95649368/0c6a22e4-fcb4-4bb6-a80f-19ca9ba79d1a)

**Reply by ghost** [reply]
از اخرین نسخه کلش استفاده کنید حداقل نسخه مجاز 2023.04.13 میتونید [اینجا ](https://github.com/Dreamacro/clash/releases/tag/premium)دانلود کنید یا از هسته کلش متا استفاده کنید.

**Comment by bitcoinvps** [comment]
من الگوی شما را در اسکریپت نصب مرزبان خود قرار داده‌ام. حالا کاربران می‌توانند با چند کلیک از کار شما لذت ببرند. marzban-ezpz

**Comment by Armania1997** [comment]
![c](https://github.com/Gozargah/Marzban/assets/72545851/426d948f-1a81-45d0-8502-ab061d44bbb7)
من فعال کردم، اما وقتی پروفایل رو میخوام اد کنم رو کلش متا، این ارور رو میده

**Comment by gorbesiah** [comment]
سلام 
ممنون بابت زحماتتون
فقط سوالی که برام پیش اومده اینه که این اسکریپت برای کلش هست یا کلش متا؟ از این جهت میپرسم که برام مهمه حتما vless پشتیبانی کنه
